### ISSUE TITLE
Midnight label issue

### ISSUE THREAD

**Issue by bisonfoutu** [kuhnroyal/mat-datetimepicker#38]
According to the official Android timepicker, the hour label for displaying midnight should be 00.
It's currently 12, this can be disturbing for users to find out which hour to select, as noon and midnight are displayed with the same value.

**Comment by kuhnroyal** [comment]
This is actually very interesting, there are so many different layouts out there.

**Comment by bushybuffalo** [comment]
Well, I since figured I did not need a timepicker in my app anyway, so I cannot currently check this out, but I clearly remember seeing two hours labelled "12" on the timepicker, and found it so disturbing that I decided to create this issue.

Below is a screenshot of the native Android timepicker I just took on my phone. I believe it should set the standard. 

![material timepicker](https://user-images.githubusercontent.com/3205090/43392494-1c9326a8-93f4-11e8-83ef-443992c501ad.jpg)
